mcsema/BC/Function.cpp

```diff
@@ -73,7 +73,10 @@ DEFINE_bool(check_pc_at_breakpoints, false,
             "Check whether or not the emulated program counter is correct at "
             "each injected 'breakpoint' function. This is a debugging aid.");
 
+#define PERSONALITY_FUNCTION  "__gxx_personality_v0"
```

Comment:
Make this a global `const char *` or `std::string`, or even a command-line flag. I don't like using macros for these kinds of constants.

Reply:
Alternatively, perhaps have a function that returns the personality function, choosing it based on the target OS? Does macOS use the gxx personality function?